common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1583,3 +1583,19 @@ def set_user_partitions_for_scheme(self, partitions, scheme):
             if p.scheme != scheme
         ]
         self.user_partitions = other_partitions + partitions  # pylint: disable=attribute-defined-outside-init
+
+    @property
+    def can_toggle_self_paced(self):
```

Comment:
Can this method be renamed to `can_toggle_course_pacing`?  I prefer that to the method having a preference to a particular pace-type